CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### HEAD
 
+### 0.13.0 - 31.01.2019
```

Comment:
I think i need to do another version here after merging. so you can skip this line for now.